release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -118,6 +118,40 @@ Limits] for {product-title} {product-version} is now available.
 Use the link:https://access.redhat.com/labs/ocplimitscalculator/[{product-title}
 Limit Calculator] to estimate cluster limits for your environment.
 
+[id="ocp-4-3-storage"]
+=== Storage
+
+[id="ocp-4-3-iscsi"]
+==== iSCSI
+
+Persistent volumes and raw block volumes using iSCSI, previously in Technology
+Preview, is now fully supported in {product-title} 4.3.
```

Comment:
The _is_ is tripping me up here. 
_Persistent volumes and raw block volumes using iSCSI, previously in Technology Preview, are now..._
_Using iSCSI with persistent volumes and raw block volumes using iSCSI, previously in Technology
Preview, is now..._
??

Reply:
Ah yes, good catch. I had two separate sentences, then combined them. That should be _are_ not _is_ .